packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -46,8 +54,14 @@ const Conversation = WebexPlugin.extend({
    * @returns {String} url of the conversation
    */
   getUrlFromClusterId({cluster = 'us', id} = {}) {
-    const clusterId =
-      cluster === 'us' ? DEFAULT_CLUSTER : cluster;
+    let clusterId =
+      cluster === 'us' ? DEFAULT_CLUSTER_IDENTIFIER : cluster;
+
+    // Determine if cluster has service name (non-US clusters from hydra do not)
+    if (clusterId.split(':').length < 4) {
+      // Add Service to cluster identifier
+      clusterId = `${cluster}:${CLUSTER_SERVICE}`;
+    }
```

Comment:
This looks great, good implementation in a good place [not in the services plugin directly, but the calling location].

As long as this remains the only service that does this weird thing, we should be good.